.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3.1.0
```

Comment:
Consider bumping to the next major version, like `uses: actions/checkout@v3` That way you don't have to care about 3.2, 3.3... you get the idea :-)